docs/userGuide/syntax/code.mbdf

```diff
@@ -32,7 +32,38 @@ To enable syntax coloring, specify a language next to the backticks before the f
 </div>
 
 ##### Line numbering
-Line numbers are provided by default. To hide line numbers, add the class `no-line-numbers` to the code block as below
+Line numbers are <tooltip content="Line numbers were provided by default in version v3.1.1 and below. To preserve the exact line numbers behavior of sites generated in previous versions, simply set the codeLineNumbers option in site.json to true">hidden by default</tooltip>. To configure the line numbers setting for the entire site (for example, set it to show by default),
+add `"codeLineNumbers": true` to the `site.json` file:
+```json {highlight-lines="8[:]"}
+{
+  "baseUrl": "/myproduct",
+  "faviconPath": "myfavicon.png",
+  "titlePrefix": "FooBar Dev Docs",
```

Comment:
```suggestion
  "titlePrefix": "FooBar Dev Docs",
```
```suggestion
// ...
```

Reply:
Actually meant to collapse the above 2 lines on baseurl / favicon, to keep the snippet slightly smaller

I'll make a point of avoiding github's suggestions if its multiline 😅

Reply:
Oh ok, I think I misunderstood because multiline suggestions are indeed a bit confusing and I wasn't sure what to edit.